ara.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
How does this work on a remote libvirt host?

Reply:
Updated to properly install on local or remote host.